src/frontend/src/lib/components/navigation/NavigationItem.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+	export let selected = false;
+	export let ariaLabel: string;
+</script>
+
+<button
+	class="w-full rounded-xl p-3 text-left hover:text-blue-ribbon"
+	class:text-blue-ribbon={selected}
```

Comment:
Can we animate / make the transition between the two states, selected/not selected smooth using some tailwind class?

Reply:
Sure, what do you think? Too quick/slow?


https://github.com/user-attachments/assets/0ae71f1d-f550-4da4-abb9-e0cccf12da8a


Or you meant to slice the background from an element to the other?

Reply:
yes! make sense, but it is mostly due to the Back button that I think we will remove... I will check